tests/format/typescript/temp/temp.ts

```diff
@@ -0,0 +1 @@
+a(b?.c!);
```

Comment:
I guess you added this to debug? you can do `yarn debug test.ts`

Reply:
Sorry i will remove it. And thanks for sharing cool command.